packages/paypal-js/src/load-script.ts

```diff
@@ -14,11 +14,17 @@ export function loadScript(
     PromisePonyfill: PromiseConstructor = Promise
 ): Promise<PayPalNamespace | null> {
     validateArguments(options, PromisePonyfill);
-
     // resolve with null when running in Node or Deno
     if (typeof document === "undefined") return PromisePonyfill.resolve(null);
 
+    if (!options.clientId) {
+        throw new Error(
+            `Expected clientId in options object: ${JSON.stringify(options)}`
```

Comment:
I like how you added what the existing options were, as it will make it easier to identify where the problem can be fixed!